app/main.py

```
@@ -21,37 +22,48 @@
-        raise
+            logger.error(f"Database connection  failed: {e}")
+
+
+    worker_service = QueueConsumer(
+        queue=supabase_queue,
+        workers=getattr(settings, 'CONSUMER_WORKERS', 2),
+        poll_interval=getattr(settings, 'CONSUMER_POLL_INTERVAL', 7.0),
+        max_processing_time=getattr(settings, 'CONSUMER_MAX_PROCESSING_TIME', 1800)
+    )
 
+    _ = asyncio.create_task(worker_service.start())
```

Ruff (0.13.3)

Store a reference to the return value of `asyncio.create_task`

(RUF006)